pyinstalive/startup.py

```diff
@@ -268,13 +269,13 @@ def run():
 
     if validate_inputs(config, args, unknown_args):
```

Comment:
`pil.proxy` is a global variable, it's already accessible in the `auth` module with `pil.proxy` thus no extra argument is required to pass with `authenticate()`.

Reply:
Thanks, I didn't realize that.